src/OpenTelemetry/Metrics/AggregationTemporality.cs

```diff
@@ -14,18 +14,31 @@
 // limitations under the License.
 // </copyright>
 
+using System;
+
 namespace OpenTelemetry.Metrics
 {
-    public enum AggregationTemporality
+    [Flags]
+    public enum AggregationTemporality : byte
     {
         /// <summary>
         /// Cumulative.
         /// </summary>
-        Cumulative = 1,
+        Cumulative = 0b1,
 
         /// <summary>
         /// Delta.
         /// </summary>
-        Delta = 2,
+        Delta = 0b10,
+
+        /// <summary>
+        /// Neither.
+        /// </summary>
+        Neither = 0b0,
```

Comment:
I feel like `None` and `All` are more common from what I have encountered in .NET but I don't object to `Neither` and `Both`.

Reply:
I would prefer None/All (that's common in C++), let me send a PR and get feedback from folks.

Reply:
#2355

Reply:
Neither also implies the set of options is only of two things

Reply:
https://github.com/open-telemetry/opentelemetry-dotnet/pull/2357/files#r709698960